govuk-social-providers/login/login.ftl

```diff
@@ -1,3 +1,4 @@
+<#ftl output_format='HTML'>
```

Comment:
Do we need this line? It isn’t used in any of Keycloak’s own templates (they only use `<#ftl output_format="plainText">` for their e-mail templates), and we haven’t used it anywhere else.

Reply:
thank you kindly ... was let by accident .. .. removed